lagunita/lms/templates/static_templates/honor.html

```diff
@@ -1,51 +1,30 @@
 <%page expression_filter="h"/>
 <%!
   from django.utils.translation import ugettext as _
+  from edxmako.shortcuts import marketing_link
 %>
-<%inherit file="../main.html" />
-<%namespace name='theme_settings' file='/theme-settings.html'/>
-<%block name="pagetitle">${_('Honor Code')}</%block>
 <%
-  theme_vars = theme_settings.get_theme_vars()
-  effective_date = theme_vars['effective_date']
+  honor_link = marketing_link('HONOR')
 %>
+<%inherit file="../main.html" />
+<%block name="pagetitle">${_('Honor Code')}</%block>
+<%block name="headextra">
+    <%!
+      from edxmako.shortcuts import marketing_link
+    %>
+    <%
+      honor_link = marketing_link('HONOR')
+    %>
+    <meta charset="UTF-8">
```

Comment:
Maybe link to the stack overflow post from which we cribbed this?
Make it a template/django/mako comment, not html.

Reply:
https://stackoverflow.com/a/5411601